easybuild/easyconfigs/b/binutils/binutils-2.25-GCC-4.9.2-binutils-2.25.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'binutils'
+version = '2.25'
+
+homepage = 'http://directory.fsf.org/project/binutils/'
+description = "binutils: GNU binary utilities"
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+toolchain = {'name': 'GCC', 'version': '4.9.2-binutils-%(version)s'}
+
+dependencies = [('zlib', '1.2.8')]
```

Comment:
No, not really. See https://github.com/hpcugent/easybuild-easyconfigs/tree/master/easybuild/easyconfigs/z/zlib (there's an easyconfig for both zlib 1.2.7 and 1.2.8 for `intel/2015a`, for example).

And there's a difference linking it *hard* (by including it, directly or indirectly, in the toolchain), and just having a policy to stick with a version.